mu4/notation/internal/excerptnotation.cpp

```diff
@@ -41,7 +45,35 @@ ExcerptNotation::~ExcerptNotation()
     setScore(nullptr);
 }
 
-Ms::Excerpt* ExcerptNotation::excerpt() const
+void ExcerptNotation::setExcerpt(Ms::Excerpt* excerpt)
+{
+    m_excerpt = excerpt;
+    setScore(m_excerpt->partScore());
+    setMetaInfo(m_metaInfo);
+}
+
+Meta ExcerptNotation::metaInfo() const
+{
+    return isInited() ? Notation::metaInfo() : m_metaInfo;
```

Comment:
Strange condition. If initialized then should we return local data?

Reply:
If notation was inited with default constructor then it has no score. We use local data in this case